platforms/dotnet/getDotNetSdk.sh

```diff
@@ -47,7 +47,12 @@ getDotNetCoreSdk() {
 		if [ -z "$downloadUrl" ]; then
 			# Use default download url file
 			downloadUrl="https://dotnetcli.blob.core.windows.net/dotnet/Sdk/$sdkVersion/dotnet-sdk-$sdkVersion-linux-x64.tar.gz"
+		elif  [ "$downloadUrl" == *"dotnet-private"* ]; then
```

Comment:
I think you need to wrap this elif in double brackets `[[  * ]]` instead of `[ * ]`, as they have different behavior
![image](https://user-images.githubusercontent.com/107068277/199100204-908ea6f6-1c9f-4223-bfa1-822dbdc8663f.png)